common/include/pcl/PCLPointCloud2.h

```diff
@@ -18,9 +18,7 @@ namespace pcl
 
   struct PCL_EXPORTS PCLPointCloud2
   {
-    PCLPointCloud2 () : height (0), width (0), 
-                     is_bigendian (false), point_step (0), row_step (0),
-                     is_dense (false)
+    PCLPointCloud2 ()
```

Comment:
Any way we can make this ` = default;` too?

Reply:
let me try